tests/cephadm/test_admin_add_labels.py

```diff
@@ -0,0 +1,64 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def _add_host_label(instance):
+    """
+    Method add multiple labels to cephadmin node
+    Args:
+        instance: cluster instance
+    Returns:
+        interger
```

Comment:
like this `1 -> Fail, 0 -> Pass`.

Reply:
@sunilangadi2 
Added as mentioned above